packages/ListBox/CHANGELOG.md

```diff
@@ -6,11 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [1.0.1]
+- Fix accessiblity issue so options are now focusable and read to screen reader technology
+- Add Toast component as depedency
+- Rename instances of Listbox to ListBox
```

Comment:
nit: if this only affects the documentation, I don't think it needs to go in the CHANGELOG